boa/src/builtins/json/mod.rs

```diff
@@ -53,7 +54,7 @@ impl Json {
                     Some(reviver) if reviver.is_function() => {
                         let mut holder = Value::new_object(None);
                         holder.set_field("", j);
-                        Self::walk(reviver, ctx, &mut holder, Value::from(""))
+                        Self::walk(reviver, ctx, &mut holder, &"".into())
```

Comment:
Just for my understanding, what does `&"".into()` do here? Do we have a more explicit way of doing the same?

Reply:
this is equivalent to `&PropertyKey::from("")`, if this is more readable I can change it

Reply:
I think it is, yes :)